low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -53,7 +53,7 @@ for group in "${!monitor[@]}"; do
     ${CONSUL_PATH}/consul kv put sspl/config/"${group}"/monitor "${monitor["${group}"]}"
 done
 
-file_store_config_path="/opt/seagate/${PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
+file_store_config_path="/opt/seagate/${PRODUCT_FAMILY}/sspl/sspl_test/sspl_tests.conf"
```

Comment:
again, why removing conf folder, we need this. avoid removing conf folder wherever you are doing in patch

Reply:
Reverted back to the old path. Done